twisher/src/main/java/posts/Filter.java

```diff
@@ -0,0 +1,24 @@
+package posts;
+
+import java.util.Date;
+
+public class Filter {
```

Comment:
Can't figure out, why is this named `Filter`. Please think thoroughly of the class names. A name must explicitly tell what a class is doing